src/contracts/FocusMode/RedirectStrategyInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Contracts\AdminUi\FocusMode;
+
+interface RedirectStrategyInterface
+{
+    public function supports(string $route): bool;
```

Comment:
Route itself is rather limiting. Shouldn't the whole route information be provided here? This may include `_controller`/`siteaccess` that has been resolved for it, and so on.

Reply:
I don't think creating an object from a specific route would be convenient in my case. What about passing an optional parameter like `array $parameters = []` that would give integrators some flexibility when it comes to distinguish what strategy is supported?

Reply:
I mean you already have an array of route data coming from URL Matcher `match` method, you're just unpacking it and leaving only `_route`.

Reply:
https://github.com/ibexa/admin-ui/pull/1065/commits/989b539431d4fa2879c7530fcb4e8a2765c81299